cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1376,20 +1386,39 @@ private KafkaVersionChange getKafkaVersionChange(StatefulSet kafkaSts) {
          */
         Future<ReconciliationState> zkScaleUpInSmallSteps() {
```

Comment:
Should this be named `scaleInSmallSteps` now?

Reply:
Renamed it to `zkScalingSetup`